src/zenml/exceptions.py

```diff
@@ -44,18 +44,24 @@ def __init__(
 
 
 class InitializationException(ZenMLBaseException):
-    """Raised when an error occurred during initialization of a ZenML
-    repository."""
+    """
```

Comment:
Not sure how we should actually format these cases: If we decide to strictly follow our coding guidelines (https://google.github.io/styleguide/pyguide.html#381-docstrings) it would mean having a one line summary followed by a more detailed explanation if necessary.

I don't actually propose following them so strictly, to me it would be okay if we're consistent with our own rules. I think most our other classes start the docstring in the same as the first `"""`,  so we should either change it everywhere or nowhere I feel

Reply:
+1

Reply:
Fixed this

Reply:
I'm surprised that our reformatter doesn't deal with this? does black not have an opinionated way of formatting docstrings?

btw PEP8 wants docstrings even narrower than normal code, 71 columns to be precise :o

Reply:
There is https://pypi.org/project/style-doc/ which claims to be the black for docstrings - I tried it in its standard configuration but its too aggressive. If we want to automatically format our docstrings we would have to spend some time to finetune it.